docs/deployment/breaking-changes/6.2-promote-to-indicator.md

```diff
@@ -0,0 +1,29 @@
+# Change to the observable "promote" API
+
+!!! info ""
+
+    * **Deprecated in**: `OpenCTI 6.2`
+    * **Removed in**: `OpenCTI 6.5`
```

Comment:
it's a bit strange to be on the future, but I think if we put "sill be removed in" we are going to forget to change in 6.5+